### DISCUSSION TITLE
Raspberry PI

### DISCUSSION THREAD

**Discussion by AlexCastro-Prog** [riclolsen/json-scada#47] [Q&A] [answered]
How can I start installing on Raspberry PI 4 ARM64?

**Comment by riclolsen** [comment, marked as answer]
Currently, there are no specific instructions for this platform.

You should follow the generic manual installation requirements, look for specific instructions at each project's site

    https://github.com/riclolsen/json-scada/blob/master/docs/install.md#manual-installation

Then clone the repo and try the platform-linux/build.sh script.

Use Supervisor to run the processes and drivers you need.
